### PR TITLE
Refactor PostExecute() method

### PR DESCRIPTION
**Pull request by php-coder**
Extracts 6 methods from `PostExecute()` method to make it shorter, readable and easy to modify.

PTAL @bparees

### CODE REVIEW

File: pkg/build/strategies/sti/sti.go [hunk 1]
```diff
@@ -272,82 +272,114 @@ func mergeLabels(newLabels, existingLabels map[string]string) map[string]string
...
-		builder.result.Messages = builder.callbackInvoker.ExecuteCallback(builder.config.CallbackURL,
-			builder.result.Success, resultLabels, builder.result.Messages)
+func (builder *STI) invokeCallbackUrl(resultLabels map[string]string) {
+	if builder.config.CallbackURL == "" {
+		return
 	}
 
-	return nil
+	builder.result.Messages = builder.callbackInvoker.ExecuteCallback(builder.config.CallbackURL,
+		builder.result.Success, resultLabels, builder.result.Messages)
```
Comment (bparees) [inline pkg/build/strategies/sti/sti.go]: if len(builder.config.CallbackURL)>0 {
  do stuff
}

Comment (bparees) [inline pkg/build/strategies/sti/sti.go]: i guess the general rule is to do what you did to minimize indenting, but in this case it's just more code.

**Comment by bparees** [comment]
one debatable nit and needs squash, otherwise lgtm.

**Comment by bparees** [comment]
[test]

**Comment by php-coder** [comment]
@bparees Ready to merge.

**Comment by bparees** [comment]
[merge]

**Pull request merged**